app/models/event.rb

```diff
@@ -224,19 +233,26 @@ def calculate_progress
   end
 
   ##
-  # Returns end of the event
+  # Returns the room in which the event is scheduled
+  #
+  def scheduled_room
```

Comment:
Why do we need to wrap a simple attribute call in a function?
Same for ``scheduled_start_time``